cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_ionic_client_app/templates/src/assets/i18n/es.json.ftl

```diff
@@ -1,67 +1,67 @@
-{
-  "buttons": {

```

Comment:
@maybeec do you know why is this file being issued as modified? Is it related to line endings problem similar to the ones Sanjay faced?